docker-compose_production_bind.yml

```diff
@@ -289,6 +289,37 @@ services:
         target: /home/mig/state/mig_system_run
     command: /app/docker-entry.sh -k
 
+  migrid-lustre-quota:
+    container_name: migrid-lustre-quota
+    image: ucphhpc/migrid${CONTAINER_TAG}
+    network_mode: "host"
+    environment:
+      TZ: ${TZ}
+    depends_on:
+      - migrid
+    cap_add:
+      - ALL
```

Comment:
Can we reduce this to only the required perms please?
https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities